client/components/AuthForm-styled.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
```

Comment:
You also never want to have a component that is a copy of another component but styled. _Everything_ in the main branch should be in a 100% completed and working state